docs/source/examples/Catch that asteroid!.ipynb

```diff
@@ -0,0 +1,890 @@
+       "<script async src=\"//platform.twitter.com/widgets.js\" charset=\"utf-8\"></script>\n"
+      ],
+      "text/plain": [
+       "<IPython.core.display.HTML object>"
+      ]
+     },
+     "execution_count": 2,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "HTML(\"\"\"\n",
+    "<blockquote class=\"twitter-tweet\" data-lang=\"en\"><p lang=\"es\" dir=\"ltr\">Sobre <a href=\"https://twitter.com/hashtag/Florence?src=hash\">#Florence</a> y los demás asteroides (ref Harris et al 2015 <a href=\"https://t.co/YfLdhEiqJb\">https://t.co/YfLdhEiqJb</a>).  Hay que hacer mucho más <a href=\"https://t.co/uqj5LabQLo\">https://t.co/uqj5LabQLo</a> <a href=\"https://t.co/NBHFAuFQAA\">pic.twitter.com/NBHFAuFQAA</a></p>&mdash; Pedro Duque (@astro_duque) <a href=\"https://twitter.com/astro_duque/status/903241796477743104\">August 31, 2017</a></blockquote>\n",
+    "<script async src=\"//platform.twitter.com/widgets.js\" charset=\"utf-8\"></script>\n",
+    "\"\"\")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/html": [
+       "\n",
+       "<blockquote class=\"twitter-tweet\" data-lang=\"en\"><p lang=\"es\" dir=\"ltr\">La <a href=\"https://twitter.com/esa_es\">@esa_es</a> ha preparado un resumen del asteroide <a href=\"https://twitter.com/hashtag/Florence?src=hash\">#Florence</a> 😍 <a href=\"https://t.co/Sk1lb7Kz0j\">pic.twitter.com/Sk1lb7Kz0j</a></p>&mdash; AeroPython (@AeroPython) <a href=\"https://twitter.com/AeroPython/status/903197147914543105\">August 31, 2017</a></blockquote>\n",
+       "<script async src=\"//platform.twitter.com/widgets.js\" charset=\"utf-8\"></script>\n"
+      ],
+      "text/plain": [
+       "<IPython.core.display.HTML object>"
+      ]
+     },
+     "execution_count": 3,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "HTML(\"\"\"\n",
+    "<blockquote class=\"twitter-tweet\" data-lang=\"en\"><p lang=\"es\" dir=\"ltr\">La <a href=\"https://twitter.com/esa_es\">@esa_es</a> ha preparado un resumen del asteroide <a href=\"https://twitter.com/hashtag/Florence?src=hash\">#Florence</a> 😍 <a href=\"https://t.co/Sk1lb7Kz0j\">pic.twitter.com/Sk1lb7Kz0j</a></p>&mdash; AeroPython (@AeroPython) <a href=\"https://twitter.com/AeroPython/status/903197147914543105\">August 31, 2017</a></blockquote>\n",
+    "<script async src=\"//platform.twitter.com/widgets.js\" charset=\"utf-8\"></script>\n",
+    "\"\"\")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%matplotlib inline\n",
+    "import matplotlib.pyplot as plt\n",
+    "plt.style.use(\"pybonacci\")\n",
```

Comment:
I would remove this.

Reply:
Yes, you're right... I opened an issue years ago to ship custom styles in matplotlib:

https://github.com/matplotlib/matplotlib/issues/4781

But until it's implemented, the only way is pointing the users to a URL and making them download the file in a specified folder... A complete mess. I will remove it (even though I love my color cycle...)